docs/custom_promise_types/password-store-sync.cf

```diff
@@ -0,0 +1,22 @@
+promise agent git
```

Comment:
@olehermanse not sure what to do with an example. How to manage putting a few examples in here and getting them to "work easy" for new users? Or just leave out the example .cf now that I have added a complete enough example to the python module?

Reply:
This example looks fine to me.

Reply:
fixed up based on my refactoring.